packages/react-native/src/Authenticator/Defaults/VerifyUser/__tests__/VerifyUser.spec.tsx

```diff
@@ -104,11 +103,4 @@ describe('VerifyUser', () => {
     expect(handleSubmit).toBeCalledTimes(1);
     expect(handleSubmit).toBeCalledWith({ unverifiedAttr: 'email' });
   });
-
-  it('renders as expected when isPending is true', () => {
```

Comment:
Why did we remove this test? How are handling the `isPending` state?

Reply:
It's noted below, but removed the `Verifying...` text to align with the other `Authenticator` implementations

Reply:
Ah, I see.